config.json

```diff
@@ -2013,20 +2013,48 @@
         "nugetVersion": "1.0.0.12",
         "nugetId": "Xamarin.AndroidX.Window.Extensions.Core.Core"
       },
+      {
+        "groupId": "androidx.work",
+        "artifactId": "work-gcm",
+        "version": "2.10.0",
+        "nugetVersion": "2.10.0.2",
+        "nugetId": "Xamarin.AndroidX.Work.GCM"
+      },
+      {
+        "groupId": "androidx.work",
+        "artifactId": "work-multiprocess",
+        "version": "2.10.0",
+        "nugetVersion": "2.10.0.2",
+        "nugetId": "Xamarin.AndroidX.Work.Work.Multiprocess"
```

Comment:
Let's remove the duplicate `.Work` from these 3 packages IDs.

Reply:
fixed